root/etc/cont-init.d/97-apt-get

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/with-contenv bash
+
+if ! [[ -x "$(command -v jq)" ]]; then
+    echo '-------------------------'
+    echo '| Installing jq package |'
+    echo '-------------------------'
+    apt-get update &&
+    apt-get install -y jq
+fi
```

Comment:
this file should be named `95-apt-get` and it should only have `apt-get update` like this one here: https://github.com/linuxserver/docker-mods/blob/code-server-golang/root/etc/cont-init.d/95-apt-get